packages/cli/src/WorkflowHelpers.ts

```diff
@@ -385,6 +386,113 @@ export async function getStaticDataById(workflowId: string | number) {
 	return workflowData.staticData || {};
 }
 
+// Checking if credentials of old format are in use and run a DB check if they might exist uniquely
+export async function replaceInvalidCredentials(workflow: WorkflowEntity): Promise<WorkflowEntity> {
```

Comment:
nitpick - might help readability if you break this into several functions

Reply:
thought about it as well, however the code is just so big due to the many checks, the logic is very limited and not used by anything else. Decided to rather keep it together to maintain the logical unit, happy to discuss more in depth if you want 😊